tmt/base.py

```diff
@@ -3934,7 +3936,7 @@ def _clean_workdir(self, path: Path) -> bool:
     def runs(self, id_: tuple[str, ...]) -> bool:
         """ Clean workdirs of runs """
         self.info('runs', color='blue')
-        root_path = Path(self.opt('workdir-root'))
+        root_path = Path(self.workdir_root)
```

Comment:
`Path()` may not be needed because `self.workdir_root` is already returning `Path` instances.

Reply:
Same as above, `Path()` is not needed and variable is used just once, we can pass `self.workdir_root` to `generate_runs()`.